### DISCUSSION TITLE
[Q] pins 27,14 dont analog read when using  WiFi.softAP("wifiname", "wifipassword"); in the esp32 wroom 32 module

### DISCUSSION THREAD

**Discussion by xamanjha** [mathieucarbou/ESPAsyncWebServer#141] [General]
**For questions: please go to the [discussions section](https://github.com/mathieucarbou/ESPAsyncWebServer/discussions)!**

hey all I dont understand the reason for this. Maybe these two pins are set to HIGH when using the 
WIFI.softAP() function 

because when I uncomment this line the analogRead Function is working fine not when I have this line 

well if this is something internally then I can't do anything write 

help me if there is another way 

I dont have any pins left out 

these pins would be good if are able to use 

ThankYou

**Comment by mathieucarbou** [comment]
Hello,
I think your question is in the wrong project ;-)
Here, this is a webserver, unrelated to any gpio...

**Comment by xamanjha** [comment]
https://github.com/user-attachments/assets/eb23ebe1-2638-45d4-bb1c-2e5b6e050486

see this screen recording 

when I use the WIFI.softAP function I read the analog values as 0,0 

and when I simply uncomment the line and upload then the values are being read and that too actual ones 

well I found this thing so I just posted it 

I dont know why its happening 

the most in simple words I am guessing is that these pins become useless when using WIFI.softAP

btw I am using esp32 wroom 32 

Thank you

**Reply by jobitjoseph** [reply]
ADC2 is not available when you're using the WIFI. The ADC2 is being used by the WiFi. And it is not an issue, but that's how it is being  designed. Either use anyone of the ADC1 pins or use a different SoC like the S3. Regardless as @mathieucarbou mentioned  it has nothing to do with this repo.

**Reply by mathieucarbou** [reply]
Thanks @jobitjoseph !

**Reply by xamanjha** [reply]
Thanks
